packages/components/src/tooltip/index.js

```diff
@@ -168,6 +168,7 @@ function Tooltip( { children, position, text, shortcut } ) {
 	};
 	const clearOnUnmount = () => {
 		delayedSetIsOver.cancel();
+		document.removeEventListener( 'mouseup', cancelIsMouseDown );
```

Comment:
I'm getting eslint warnings about us not using `ownerDocument` here. Maybe one for a followup?

Reply:
The attached event is using `document` already, so yeah probably a followup, and not related to this PR.